README.md

```diff
@@ -94,6 +97,183 @@ see [Developer’s guide to auth with Azure Resource Manager API](http://aka.ms/
 After creating the service principal, you should have three pieces of information, a client id (GUID), client secret
 (string) and tenant id (GUID) or domain name (string).
 
+# Azure Multiple API versions & Profiles
+
+With 0.15.0 of Azure SDK, multiple API versions and profiles are introduced. With these changes, each individual gem
+has multiple versions of the services and several profiles. The azure sdk rollup gem also consists of several profiles. The following section provides details on the usage of multiple API versions and profiles.
+
+## Why Multiple API versions?
+
+With the previous Azure SDK, you will have access to the latest (at the time of release) versions of our services. With 
+each new release, the versions may get updated. But, if you would like to continue using the old versions of some 
+services, such option was not available. But, with the multiple API versions, you could use any version of service. 
+
+## What is a  Profile?
+
+A profile is a combination of different resource types with different versions from different services. Using a profile,
+will help you mix and match between various resource types.
+
+## What to use?
+* If you would like to use the latest versions of **all** the services, then the recommendation is to use the **latest** profile of the Azure SDK rollup gem.
```

Comment:
should we say `Latest` ? as it's the actual name, and have a pointer, to the file where we are defining it ? https://github.com/Azure/azure-sdk-for-ruby/blob/master/profile_specs/profiles.json#L47

Reply:
Agreed that we should say 'Latest' since this is the actual name. But, we do not want to point to the json file for 2 reasons. 1. We are going to change it soon. 2. We do not want our customers to look into our internal generation process